spec/system/admin_order_spec.rb

```diff
@@ -41,6 +41,9 @@
       expect(page).to have_content reward.title
       expect(page).to have_content reward.description
       expect(page).to have_content reward.price.to_i
+
+      click_button 'Deliver', match: :first
```

Comment:
I think we should also test the scenario, when the order is already delivered it can't be delivered again? (for example by checking, that button is not present after the delivery)

Reply:
I added changes